RecoMET/METAlgorithms/src/METSignificance.cc

```diff
@@ -93,8 +94,7 @@ metsig::METSignificance::getCovariance(const edm::View<reco::Jet>& jets,
 
    }
 
-   // calculate sumPt
-   double sumPt = 0;
```

Comment:
`double sumPt = sumPtUnclustered;`
and use `sumPt` instead of `sumPtUnclustered` in the rest of `METSignificance.cc`